emjunior258/certeasy
# feat: list certificate issuers

## Description

This PR adds the feature to list certificate issuers:
- Implements the application shell/layout
- implements empty state component

## Dependencies
- #127 

## How to test
1. Make sure you have NodeJS installed.
2. enable corepack by running the `corepack enable` command.
3. install dependencies by running the `pnpm install` command.
4. start the application by running the `pnpm dev` command.
5. Open http://localhost:5173/ in your browser and see that it lists applications, note that there is a mock server behind it.

Note: to simulate no records use the following URL containing a trap http://localhost:5173?trap=NO_REDORDS

## Screenshots 
### Empty State
#### Mobile 
![iPhone 12 Pro-1731601948928](https://github.com/user-attachments/assets/cc378cdc-ef4f-4c61-966e-07e6acdfa8ca)

#### Tablet
![iPad-1731602015106](https://github.com/user-attachments/assets/ed1af0d6-f6a9-4889-aa44-33e0510db68c)

#### Desktop
![MacBook Pro-1731602064935](https://github.com/user-attachments/assets/c81b16b8-a077-4d28-b72c-f935e4db1ebb)

### Non-Empty State
#### Mobile
![iPhone 12 Pro-1731602183647](https://github.com/user-attachments/assets/3cea6f66-3a0b-4fb9-b17a-36d14a653aa8)

#### Tablet
![iPad-1731602201996](https://github.com/user-attachments/assets/b1a3e8ed-171b-45f7-9314-44e2d9013d07)

#### Desktop
![MacBook Pro-1731602229648](https://github.com/user-attachments/assets/7cf1b73a-d7d1-4b76-9dc5-01b2ba1ff458)
